dashboard/app/controllers/admin_hoc_controller.rb

```diff
@@ -0,0 +1,66 @@
+require 'cdo/env'
+
+# The controller for reports of HOC data.
+class AdminHocController < ApplicationController
+  before_filter :authenticate_user!
+  before_action :require_admin
+  check_authorization
+
+  include LevelSourceHintsHelper
+
+  def event_signups
+    SeamlessDatabasePool.use_persistent_read_connection do
+      # Requested by Roxanne on 16 November 2015 to track HOC 2015 signups by day.
+      # Get the HOC 2014 and HOC 2015 signup counts by day, deduped by email and name.
+      # We restrict by dates to avoid long trails of (inappropriate?) signups.
+      data_2014 = DB[:forms].
+                  where('kind = ? AND created_at > ? AND created_at < ?', 'HocSignup2014', '2014-08-01', '2015-01-01').
+                  group(:name, :email).
+                  # TODO(asher): Is this clumsy notation really necessary? Is Sequel
```

Comment:
Now that you've had a bit more time to play with Sequel since you originally wrote this, do you have a satisfying answer to this question?